root/static/scripts/relationship-editor/utility/prepareHtmlFormSubmission.js

```diff
@@ -210,10 +213,20 @@ export default function prepareHtmlFormSubmission(
       }
 
       if (hasSessionStorage) {
-        window.sessionStorage.setItem(
-          'relationshipEditorState',
-          JSON.stringify(compactEntityJson(state)),
-        );
+        try {
+          window.sessionStorage.setItem(
+            'relationshipEditorChanges',
+            JSON.stringify(compactEntityJson(changes)),
+          );
+        } catch (error) {
+          /*
+           * Don't prevent the form from submitting if we encounter
+           * an error; for instance, "quota exceeded," which we can't do
+           * anything about. Exceeding the storage quota should be much
+           * less likely now that MBS-13393 is mitigated.
```

Comment:
What *would* happen if this got hit still somehow, and the form was submitted? Is there a risk of us interpreting something here as a reason to remove data?

Reply:
All of the form will be submitted normally, but if there is an error and the form reloads, the submitted relationships and/or external links won't be restored. Nothing will be removed, it'll just behave as if you loaded the edit page from scratch.